src/shared/containers/GetOrganizations.tsx

```diff
@@ -61,16 +64,31 @@ const GetOrganizations: FunctionComponent = () => {
     ) {
       dispatch(apiGetQuartzMe())
     }
+  }, [dispatch, quartzMeStatus])
 
+  useEffect(() => {
     if (
       isFlagEnabled('credit250Experiment') &&
-      quartzMeStatus === RemoteDataState.Done
+      quartzMeStatus === RemoteDataState.Done &&
+      status === RemoteDataState.Done
     ) {
-      const {accountCreatedAt = ''} = me
+      const {
+        accountType: account_type,
+        accountCreatedAt: account_created_at = '',
+      } = quartzMe
+      const {id: orgId = ''} = org
       window.dataLayer = window.dataLayer ?? []
-      window.dataLayer.push({accountCreatedAt})
+      window.dataLayer.push({
+        identity: {
+          account_type,
+          account_created_at,
+          id: meId,
+          email,
+          organization_id: orgId,
+        },
+      })
     }
-  }, [dispatch, me, quartzMeStatus])
+  }, [quartzMeStatus, status]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
If we use all of the dependencies suggested by the linter, then the `identity` gets pushed again to the `dataLayer` unnecessarily. We just want the status of the two api calls to be `RemoteDataState.Done` before executing the push.